.travis.yml

```diff
@@ -3,7 +3,7 @@ script: "script/test_all"
 sudo: false
 email: false
 before_install:
-  - gem update --system # update rubygems
+  - gem update --system 2.7.8 # update rubygems
```

Comment:
I think we should do this only on older Rubies, see rspec/rspec-dev#211 and its associated PRs